### PR TITLE
add test for rh-php72

### PR DESCRIPTION
**Pull request by remicollet**

### CODE REVIEW

File: collections/rh-php72-rh/check-version/run.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+source ${THISDIR}/../../../common/functions.sh
+source ${THISDIR}/../include.sh
+
+out=$(scl enable $ENABLE_SCLS "php -v")
+ret=$?
+
+echo "$out"|grep -o 'PHP 7\.1\.'
```
Comment (khardix) [inline collections/rh-php72-rh/check-version/run.sh]: I think that should be `7\.2\.`:

```suggestion
echo "$out"|grep -o 'PHP 7\.2\.'
```

File: collections/rh-php72-rh/include.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export INSTALL_SCLS=rh-php71
+export ENABLE_SCLS=rh-php71
+export PHP_HTTPD_PKGS="httpd24 rh-php71-php-fpm"
```
Comment (khardix) [inline collections/rh-php72-rh/include.sh]:
```suggestion
export PHP_HTTPD_PKGS="httpd24 rh-php72-php-fpm"
```

File: collections/rh-php72-rh/include.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+export HTTPD_SERVICE_NAME="httpd24-httpd rh-php71-php-fpm"
```
Comment (khardix) [inline collections/rh-php72-rh/include.sh]:
```suggestion
export HTTPD_SERVICE_NAME="httpd24-httpd rh-php72-php-fpm"
```

**Review by khardix — CHANGES_REQUESTED** [review]
LGTM, apart from the version-dependent checks :wink:

File: collections/rh-php72-rh/check-version/run.sh (hunk 1 above)
Comment (remicollet) [reply collections/rh-php72-rh/check-version/run.sh]: indeed; fixed (in a simpler way which will work for 7.3, 7.4, ...)

File: collections/rh-php72-rh/include.sh (hunk 3 above)
Comment (remicollet) [reply collections/rh-php72-rh/include.sh]: fixed...

**Review by khardix — APPROVED** [review]

**Pull request merged**